lib/elixir/lib/kernel.ex

```diff
@@ -4020,24 +4020,18 @@ defmodule Kernel do
       is allowed and its results will be evaluated at runtime. Usually
       evaluates to the name of a module.
 
+      If the expression evaluates to a list, it will emit a warning,
+      since lists of expressions have been deprecated.
+
```

Comment:
As @josevalim said, we could remove this entirely since we don't advertise the possibility of using lists here anymore (so if you're still using them because you thought they worked, you'll get a deprecation warning). Wdyt?